mne/viz/_brain/_brain.py

```diff
@@ -2654,24 +2656,26 @@ def show(self):
         """Display the window."""
         self._renderer.show()
 
-    def show_view(self, view=None, roll=None, distance=None, row=0, col=0,
-                  hemi=None, align=True):
+    @fill_doc
+    def show_view(self, view=None, roll=None, distance=None,
+                  azimuth=None, elevation=None, focalpoint=None,
+                  row=None, col=None, hemi=None, align=True):
```

Comment:
Order is still wrong, new args must go at the end or it will break people's previously working code with no warning, e.g. if someone had `brain.show_view(None, 15, 0.1, 0, 0)` it is now broken. If we really want to change the order, you at least need to do:
```
    def show_view(self, view=None, roll=None, distance=None,
                  *, azimuth=None, elevation=None, focalpoint=None,
                  row=None, col=None, hemi=None, align=True):
```
but even this will break people's code. Thus even though it's not as clean, I'd rather keep the old order:
```suggestion
    def show_view(self, view=None, roll=None, distance=None,
                  row=None, col=None, hemi=None, align=True,
                  azimuth=None, elevation=None, focalpoint=None):
```

Reply:
Ok, I figured people didn't use that many ordered arguments but your right better to be safe